wallet/wallet.c

```diff
@@ -658,6 +660,18 @@ bool wallet_can_spend(struct wallet *w, const u8 *script,
 	return false;
 }
 
+void wallet_set_keypath(struct wallet *w, u32 index, struct wally_map *map_in)
```

Comment:
could merge this wrapper code into `psbt_set_keypath` by passing `w->bip32_base` (the parent xkey) into it?

Reply:
But `psbt_set_keypath` is also called from:
- `psbt_add_keypath_to_last_output`, called from:
  - `create_close_tx
  - `replace_penalty_tx_to_us`
  - `tx_to_us`
  
 Do I need to find the parent xkey in those context's too?

Reply:
nm, `wallet_set_keypath` was only called in one place, was able to remove it and move the wrapper code there.